### PR TITLE
Add Hyundai KONA Hybrid 2020

### PR DESCRIPTION
**Pull request by haram-KONA**
<!--- ***** Add hyundai KONA Hybrid 2020 *****

**Description** [](Add Car KONA Hybrid 2020.)

**Verification** [](results: 49f3c13141b6bc87|2021-07-28--08-05-13--0)

**Route**
Route: [49f3c13141b6bc87|2021-07-28--08-05-13]

### CODE REVIEW

**Comment by haram-KONA** [comment]
Please Add a Car Hyundai KONA Hybrid 2020

File: selfdrive/car/hyundai/interface.py [hunk 1]
```diff
@@ -22,9 +22,9 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=[]):  # py
     # Most Hyundai car ports are community features for now
     ret.communityFeature = candidate not in [CAR.SONATA, CAR.PALISADE]
 
-    ret.steerActuatorDelay = 0.1  # Default delay
+    ret.steerActuatorDelay = 0.0  # Default delay
     ret.steerRateCost = 0.5
-    ret.steerLimitTimer = 0.4
+    ret.steerLimitTimer = 2.5
```
Comment (adeebshihadeh) [inline selfdrive/car/hyundai/interface.py:27]: why are these changed?

File: selfdrive/car/hyundai/values.py [hunk 2]
```diff
@@ -65,6 +66,9 @@ class Buttons:
   CAR.ELANTRA: [{
     66: 8, 67: 8, 68: 8, 127: 8, 273: 8, 274: 8, 275: 8, 339: 8, 356: 4, 399: 8, 512: 6, 544: 8, 593: 8, 608: 8, 688: 5, 790: 8, 809: 8, 897: 8, 832: 8, 899: 8, 902: 8, 903: 8, 905: 8, 909: 8, 916: 8, 1040: 8, 1056: 8, 1057: 8, 1078: 4, 1170: 8, 1265: 4, 1280: 1, 1282: 4, 1287: 4, 1290: 8, 1292: 8, 1294: 8, 1312: 8, 1314: 8, 1322: 8, 1345: 8, 1349: 8, 1351: 8, 1353: 8, 1363: 8, 1366: 8, 1367: 8, 1369: 8, 1407: 8, 1415: 8, 1419: 8, 1425: 2, 1427: 6, 1440: 8, 1456: 4, 1472: 8, 1486: 8, 1487: 8, 1491: 8, 1530: 8, 1532: 5, 2001: 8, 2003: 8, 2004: 8, 2009: 8, 2012: 8, 2016: 8, 2017: 8, 2024: 8, 2025: 8
   }],
+  CAR.KONA_HEV: [{
```
Comment (adeebshihadeh) [inline selfdrive/car/hyundai/values.py:69]: remove this. can fingerprints are deprecated

Comment (haram-KONA) [reply selfdrive/car/hyundai/values.py:69]: yes

File: selfdrive/car/hyundai/interface.py (hunk 1 above)
Comment (haram-KONA) [reply selfdrive/car/hyundai/interface.py:27]: it is my mistake  ...

**Review by haram-KONA — COMMENTED** [review]
modified my request .

**Comment by haram-KONA** [comment]
Since the Kona Hybrid vehicle is not supported by Comma OpenPilot, I am confused as I have to modify it every time I use it.
For this reason, there was an unintentional mistake in the process of making the request,
and I have corrected what you have said.


Thank you very much for your consideration.


Windows 10용 메일<https://go.microsoft.com/fwlink/?LinkId=550986>에서 보냄

보낸 사람: Adeeb ***@***.***>
보낸 날짜: 2021년 7월 29일 목요일 오전 6:58
받는 사람: ***@***.***>
참조: ***@***.***>; ***@***.***>
제목: Re: [commaai/openpilot] Add Hyundai KONA Hybrid 2020 (#21761)


@adeebshihadeh commented on this pull request.


In selfdrive/car/hyundai/values.py<https://github.com/commaai/openpilot/pull/21761#discussion_r678680502>:

> @@ -65,6 +66,9 @@ class Buttons:

   CAR.ELANTRA: [{

     66: 8, 67: 8, 68: 8, 127: 8, 273: 8, 274: 8, 275: 8, 339: 8, 356: 4, 399: 8, 512: 6, 544: 8, 593: 8, 608: 8, 688: 5, 790: 8, 809: 8, 897: 8, 832: 8, 899: 8, 902: 8, 903: 8, 905: 8, 909: 8, 916: 8, 1040: 8, 1056: 8, 1057: 8, 1078: 4, 1170: 8, 1265: 4, 1280: 1, 1282: 4, 1287: 4, 1290: 8, 1292: 8, 1294: 8, 1312: 8, 1314: 8, 1322: 8, 1345: 8, 1349: 8, 1351: 8, 1353: 8, 1363: 8, 1366: 8, 1367: 8, 1369: 8, 1407: 8, 1415: 8, 1419: 8, 1425: 2, 1427: 6, 1440: 8, 1456: 4, 1472: 8, 1486: 8, 1487: 8, 1491: 8, 1530: 8, 1532: 5, 2001: 8, 2003: 8, 2004: 8, 2009: 8, 2012: 8, 2016: 8, 2017: 8, 2024: 8, 2025: 8

   }],

+  CAR.KONA_HEV: [{

remove this. can fingerprints are deprecated

**Comment by haram-KONA** [comment]
Please~~ Modifyed All Request

**Review by haram-KONA — COMMENTED** [review]
finish  modify my request

**Comment by haram-KONA** [comment]
I'm not an expert in this field, I'm a beginner, so I don't know. What is My PR status?

**Comment by pd0wm** [comment]
new PR: https://github.com/commaai/openpilot/pull/21819
